boltdesignsystem/bolt
# Border Radius | FE | Convert components to use border radius function

## Jira

http://vjira2:8080/browse/BDS-1156

## Summary

Updated component specific CSS to use the new border-radius function.

## Details

Many components' CSS got updated, such as the following:

* blockquote
* button
* card
* chip
* code snippet
* form
* icon
* nav-priority
* navlink
* pagination
* tooltip
* video

## How to test

Run the branch locally and checked the demo pages for the list of affected components. Make sure the border radius is displaying correctly.
